.github/workflows/validate-device-config-files.yml

```diff
@@ -1,6 +1,7 @@
 name: Device configuration files
 
-on: [push]
+on:
+  - pull_request
```

Comment:
Why not both? It's possible that someone opens a PR as a branch of this repo.

Reply:
But is a PR not a PR even in the same repo?

Reply:
Yes a PR is a PR regardless of the origin. Running the CI on push would mean, that all jobs are triggered again, after the PR is merged.